Gemfile

```diff
@@ -117,7 +117,7 @@ group :check do
   # version and bump it from time to time.
   # Also, there's no support for windows
   if RUBY_VERSION >= '2.4.0' && (RUBY_PLATFORM =~ /^x86_64-(darwin|linux)/)
-    gem 'sorbet', '= 0.5.9672'
+    gem 'sorbet', '= 0.5.10201'
```

Comment:
Nothing that needs to be changed in this PR, but I'm curious if there's any reason to pick 0.5.10201 specifically, since it's a version from last August.

Reply:
It's the oldest version that has `darwin-22` packages published.
I did a conservative upgrade, just because I didn't know how much risk it would be to fully upgrade it.

Reply:
Ack, thanks for satisfying my curiosity ;)